lints/lint_rsa_not_null_param.go

```diff
@@ -0,0 +1,92 @@
+// lint_rsa_not_null_param.go
+/********************************************************************
```

Comment:
This is saying the SPKI algorithm parameter is present, but not a null. We discussed this the other day in slack, you also showed a screenshot of this.